### PR TITLE
setup skeleton views, connected with google maps api - merge

### PR DESCRIPTION
**Pull request by bel11**
setup skeleton views
connected with google maps api - can search for location using autocomplete prediction

### CODE REVIEW

**Review by bel11 — COMMENTED** [review]
autcomplete_predictor_demo

File: app/src/main/java/com/example/travelapp/MainActivity.java [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
             @Override
-            public void onClick(View v) {
-                ParseUser.logOut();
-                // go back to login screen
-                Intent i = new Intent(MainActivity.this, LogInActivity.class);
-                startActivity(i);
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                Fragment fragment;
+                switch (item.getItemId()) {
```
Comment (mbarbagallo) [inline app/src/main/java/com/example/travelapp/MainActivity.java:40]: nice using a switch statement!

File: app/src/main/res/layout/fragment_compose.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        android:layout_marginTop="30dp" />
+
+<!--    <EditText-->
+<!--            android:id="@+id/etMoreLocations"-->
+<!--            android:layout_width="200dp"-->
+<!--            android:layout_height="wrap_content"-->
+<!--            android:layout_alignParentStart="true"-->
+<!--            android:layout_marginStart="59dp"-->
+<!--            android:layout_marginTop="30dp"-->
+<!--            android:hint="Enter more locations" />-->
```
Comment (mbarbagallo) [inline app/src/main/res/layout/fragment_compose.xml:23]: nit: don't commit commented out code

**Review by mbarbagallo — APPROVED** [review]
small nits (see comments) but i'll approve and merge

**Pull request merged**